core/controllers/contributor_dashboard.py

```diff
@@ -948,8 +948,9 @@ def post(self) -> None:
         assert self.user_id is not None
         assert self.normalized_payload is not None
         language_code = self.normalized_payload['language_code']
-        user_services.update_preferred_translation_language_code(
-            self.user_id, language_code)
+        user_settings = user_services.get_user_settings(self.user_id)
```

Comment:
Should this be strict=True as well?

Reply:
Actually the default value for 'strict' param is True. Do you want me to explicitly mention strict=True ?

Reply:
If the default is True then you don't need to mention it either here or below. I just noted this because the two invocations were inconsistent with each other.

Reply:
Made it consistent with other instances.
Done.